File: myblog-server/src/main/java/com/site/blog/model/entity/BlogConfig.java
package com.site.blog.model.entity;

import java.io.Serial;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import java.io.Serializable;
import java.time.LocalDateTime;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.*;
import lombok.experimental.Accessors;

/**
 * <p>
 * 
 * </p>
 *
 */
@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BlogConfig implements Serializable {

@Serial
private static final long serialVersionUID=1L;
@TableId(type = IdType.ASSIGN_ID)
private String id;
    /**
     * 字段名
     */

    private String configCode;

    /**
     * 配置名
     */

    private String configName;

    /**
     * 配置项的值
     */

    private String configValue;

    /**
     * 创建时间
     */
    @TableField("create_time")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime createTime;

    /**
     * 修改时间
     */
    @TableField("update_time")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime updateTime;


}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/model/entity/BlogConfig.kt
package com.site.blog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableId
import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime
 
data class BlogConfig  (
    @TableId(type = IdType.ASSIGN_ID)
    var  id: String? = null,

    /**
     * 字段名
     */
    var  configCode: String  = "",

    /**
     * 配置名
     */
    var  configName: String  = "",

    /**
     * 配置项的值
     */
    var  configValue: String? = null,

    /**
     * 创建时间
     */
    @TableField("create_time")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var  createTime: LocalDateTime? = null,

    /**
     * 修改时间
     */
    @TableField("update_time")
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    var  updateTime: LocalDateTime? = null)

     
 